mods/e2140/content/ed/infantry/a04/rules.yaml

```
@@ -0,0 +1,22 @@
+ed_infantry_a04:
+	Inherits: ^EdInfantry
+	Tooltip:
+		Name: A04 Android
+	Valued:
+		Cost: 100
+	Buildable:
+		IconPalette:
+		Queue: Infantry.ED
+	Health:
+		HP: 20
+	RevealsShroud:
+		Range: 2c896
+	#TODO Add mines
+	EnhancedEncyclopedia:
+		Category: ED - Infantry
+		Order: 4
+		Animation: DATABASE/A04.FLC
+		Title: A04 Android
```

Use original name *Android A04*:

```suggestion
		Title: Android A04
```